website/docs/add-secure-apps/providers/saml/index.md

```diff
@@ -14,20 +15,41 @@ Default fields are exposed through auto-generated Property Mappings, which are p
-- When the request asks for `urn:oasis:names:tc:SAML:2.0:nameid-format:WindowsDomainQualifiedName`, the NameID will be set to the user's UPN. This is also set by the LDAP source, and also falls back to the persistent identifier.
-- When the request asks for `urn:oasis:names:tc:SAML:2.0:nameid-format:transient`, the NameID will be set based on the user's session ID.
-- When the request asks for `urn:oasis:names:tc:SAML:1.1:nameid-format:emailAddress`, the NameID will be set to the user's email address.
+| Value                                 | NameID Format                                                          | Fallback              |
+| ------------------------------------- | ---------------------------------------------------------------------- | --------------------- |
+| Hashed user ID                        | `urn:oasis:names:tc:SAML:2.0:nameid-format:persistent`                 | N/A                   |
+| User's `distinguishedName` attribute¹ | `urn:oasis:names:tc:SAML:2.0:nameid-format:X509SubjectName`            | Persistent identifier |
+| User Principal Name (UPN)¹            | `urn:oasis:names:tc:SAML:2.0:nameid-format:WindowsDomainQualifiedName` | Persistent identifier |
+| User's session ID                     | `urn:oasis:names:tc:SAML:2.0:nameid-format:transient`                  | N/A                   |
+| User's email address                  | `urn:oasis:names:tc:SAML:1.1:nameid-format:emailAddress`               | N/A                   |
 
-    :::warning
-    Keep in mind that with the default settings, users are free to change their email addresses. As such it is recommended to use `urn:oasis:names:tc:SAML:2.0:nameid-format:persistent`, as this cannot be changed.
-    :::
+¹ These attributes are typically set by LDAP sources.
+
+:::warning
+Keep in mind that with the default settings, users are free to change their email addresses. As such it is recommended to use `urn:oasis:names:tc:SAML:2.0:nameid-format:persistent`, as this cannot be changed.
+:::
+
+## Single Logout Service URL
+
+The **Single Logout Service URL** is the service provider's endpoint to which authentik sends logout requests. If you want to enable [SAML IDP-initiated single logout](./IDP-initiated-single-logout.md), this field is required.
+
+1. In your SAML provider, set the **SLS URL** field to your service provider's logout endpoint
+2. Choose the appropriate **SLS Binding**:
+    - **Redirect** - Uses HTTP redirects (front-channel only)
+    - **POST** - Supports both front-channel and back-channel
+
+:::tip
+When using POST binding, you can enable **Backchannel Post Logout** for server-to-server logout. This ensures users are logged out even when their session is administratively terminated.
```

Comment:
"... even when..." what does this mean? I'd expect that if a user;s session is administr